drivers/volume/volume.go

```diff
@@ -136,8 +136,13 @@ type Driver interface {
 
 var (
 	volDrivers = make(map[string]Driver)
+	// StorageProvisioner to be used to store name of the storage provisioner
+	StorageProvisioner string
```

Comment:
Change the type to `StorageProvisionerType`

Reply:
Fixed.